code/modules/events/_event.dm

```diff
@@ -130,15 +129,14 @@
 /datum/round_event/proc/start()
 	return
 
-//Called after start()
-//Lets the ghosts know that the event has started, and provides a follow link to a mob if possible
+//Called after something followable has been spawned by an event
+//Provides ghosts a follow link to an atom if possible
 //Only called once.
-/datum/round_event/proc/announce_to_ghosts()
+/datum/round_event/proc/announce_to_ghosts(atom/atom_of_interest)
+	message_admins("meme")
```

Comment:
Ahem

Reply:
i should really start looking at my own code